style.css

```diff
@@ -31,54 +32,214 @@ html{
 	grid-row: 2;
 	grid-column: 1/4;
 	max-width: 100%;
+	text-align: center;
 }
 
 .box_3{
 	background-color: #fbfcf8;
-	border-bottom: 3px solid #b8860b;
-	border-top: 3px solid #b8860b;
 	grid-row: 3;
 	grid-column: 1/4;
-	max-width: 100%;
+	width: 100%;
 }
 
 .box_4{
 	background-color: #fbfcf8;
-	border-left: 1px solid #888888;
-	border-right: 1px solid #888888;
 	grid-row: 4;
 	grid-column: 1/4;
-	width: 90%;
-	margin: 0 auto;
-	padding: 25px;
 }
 
 .box_5{
-	background-color: #fbfcf8;
-	border-bottom: 3px solid #b8860b;
-	border-top: 1px solid #888888;
+	background-color: #f3f3f3;
 	grid-row: 5;
 	grid-column: 1/4;
 	max-width: 100%;
+	text-align: center;
+	padding: 10px;
+}
+
+
+/*---------- Primary Content Box (4) CSS ----------*/
+.box_4_content {
+	width: 90%;
+	background-color: #fbfcf8;
+	margin: auto;
+	padding: 10px 30px	;
+	height: auto;
+}
+
+/*.content {
+	margin: 0 0 25px;
+	height: auto;
+	text-align: left;
+}*/
+
+/*---------- Homepage Only CSS----------*/
+/*Title Container*/
+.titlecontainer {
+	margin: 2.5vw;
+}
+.titlehome {
+	margin-bottom: 5px;
 }
 
-div.polaroidG{

```

Comment:
I've completely reworked the polaroid card system to make it more easily manageable and cut back on extra code.